Exception thrown when adding a new note

**Steps to reproduce**

Steps to reproduce the behavior:
1. Open a space
2. Go to application Notes and create a new note
3. Fill it with text then save
4. Check the server logs, there is an error about some missing class : 
```
2022-09-27 17:18:14,164 | [1;31mERROR[0;39m | Job portal:ElasticSearch.ESBulkIndexer threw an unhandled Exception:  [32m[org.quartz.core.JobRunShell[0;39m[36m<ExoScheduler_Worker-3>[0;39m[32m][0;39m 
java.lang.NoClassDefFoundError: org/jsoup/Jsoup
	at org.exoplatform.wiki.utils.Utils.html2text(Utils.java:709)
	at org.exoplatform.wiki.jpa.search.WikiPageIndexingServiceConnector.create(WikiPageIndexingServiceConnector.java:129)
	at org.exoplatform.commons.search.es.client.ElasticContentRequestBuilder.getCreateDocumentRequestContent(ElasticContentRequestBuilder.java:145)
	at org.exoplatform.commons.search.index.impl.ElasticIndexingOperationProcessor.processCUD(ElasticIndexingOperationProcessor.java:337)
	at org.exoplatform.commons.search.index.impl.ElasticIndexingOperationProcessor.processBulk(ElasticIndexingOperationProcessor.java:203)
	at org.exoplatform.commons.search.index.impl.ElasticIndexingOperationProcessor.process(ElasticIndexingOperationProcessor.java:152)
	at org.exoplatform.commons.search.job.BulkIndexingJob.execute(BulkIndexingJob.java:41)
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202)
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)
Caused by: java.lang.ClassNotFoundException: org.jsoup.Jsoup
	... 9 common frames omitted

```

**Current behaviour**

The exception is thrown and the page is not indexed in elastic search

**Expected behavior**

The page is well indexed in Elasticsearch, and no errors in logs

**(optional) Affected environments**  
Please complete the following information: 
 - OS: All
 - Browser : N/A
 - Version N/A
 - Device: N/A

--- a/notes-packaging/src/main/assemblies/notes-addon-package.xml
+++ b/notes-packaging/src/main/assemblies/notes-addon-package.xml
@@ -43,6 +43,7 @@
         <include>${project.groupId}:*:jar</include>
         <include>commons-configuration:commons-configuration:jar</include>
         <include>org.suigeneris:jrcs.rcs:jar</include>
+        <include>org.jsoup:jsoup:jar</include>
       </includes>
       <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
       <useProjectArtifact>false</useProjectArtifact>

